components/ArtPieces/ArtPieces.js

```diff
@@ -0,0 +1,26 @@
+import styled from "styled-components";
+import ArtPiecePreview from "../ArtPiecePreview/ArtPiecePreview";
+export default function ArtPieces({ pieces }) {
+  return (
+    <>
```

Comment:
these fragments are not needed (only if you want to return two or more elements side by side.